app/models/scripts/dataCreation.js

```diff
@@ -1,6 +1,6 @@
+const orm = require('../../orm');
 
 exports.execute = (db) => {
-
+  // orm.models.user.create({ firstName: 'test', lastName: 'wolox', password: '123456789', email: 'test@wolox.com.ar' }).then();
```

Comment:
We should try figure out why this isn't working right, so we can add it in this PR.